
ResNetMTL_Split
Dataset: Nyuv2 | Training Task: Semantic + Depth + Normals | Primary Task: Semantic + Depth + Normals in Multi-task / Auxiliary Learning Mode with RESNET
Applying Multi-task Methods: Weighting-based: Equal + Gradient-based: None
STEP. Loading datasets...
Data sanity check. RGB.shape: torch.Size([4, 3, 288, 384]),	Depth.shape torch.Size([4, 1, 288, 384]),    	Semantic.shape torch.Size([4, 288, 384]),	Normals.shape torch.Size([4, 3, 288, 384])
  0%|                                                                                                                                                                                                  | 0/654 [00:00<?, ?it/s]train_model.py:305: RuntimeWarning: invalid value encountered in true_divide
  depth = (depth - np.min(depth)) / ((np.max(depth) - np.min(depth))) + 1e-6














 60%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████▋                                                                          | 390/654 [00:52<00:35,  7.37it/s]
Traceback (most recent call last):
  File "train_model.py", line 402, in <module>
    for multitaskdatatest in tqdm(test_loader):
  File "/home/dim/anaconda3/envs/pytorchenv/lib/python3.7/site-packages/tqdm/std.py", line 1205, in __iter__
    self.update(n - last_print_n)
  File "/home/dim/anaconda3/envs/pytorchenv/lib/python3.7/site-packages/tqdm/std.py", line 1256, in update
    self.refresh(lock_args=self.lock_args)
  File "/home/dim/anaconda3/envs/pytorchenv/lib/python3.7/site-packages/tqdm/std.py", line 1361, in refresh
    self.display()
  File "/home/dim/anaconda3/envs/pytorchenv/lib/python3.7/site-packages/tqdm/std.py", line 1509, in display
    self.sp(self.__str__() if msg is None else msg)
  File "/home/dim/anaconda3/envs/pytorchenv/lib/python3.7/site-packages/tqdm/std.py", line 350, in print_status
    fp_write('\r' + s + (' ' * max(last_len[0] - len_s, 0)))
  File "/home/dim/anaconda3/envs/pytorchenv/lib/python3.7/site-packages/tqdm/std.py", line 343, in fp_write
    fp.write(_unicode(s))
  File "/home/dim/anaconda3/envs/pytorchenv/lib/python3.7/site-packages/tqdm/utils.py", line 145, in inner
    return func(*args, **kwargs)
